extensions/wikia/Search/classes/IndexService/DefaultContent.php

```diff
@@ -60,40 +67,39 @@ public function execute() {
 		$pageId = $service->getCanonicalPageIdFromPageId( $this->currentPageId );
 
 		// we still assume the response is the same format as MediaWiki's
-		$response   = $service->getParseResponseFromPageId( $pageId );
+		$response = $service->getParseResponseFromPageId( $pageId );
 
 		// ensure the response is an array, even if empty.
-		$response   = $response == false ? array() : $response;
-		$titleStr   = $service->getTitleStringFromPageId( $pageId );
+		$response = $response == false ? [] : $response;
+		$titleStr = $service->getTitleStringFromPageId( $pageId );
 
-		$this->pushNolangTxt( $titleStr )
-		     ->pushNolangTxt( preg_replace( '/([[:punct:]])/', ' $1 ', $titleStr ) );
+		$this->pushNolangTxt( $titleStr )->pushNolangTxt( preg_replace( '/([[:punct:]])/', ' $1 ', $titleStr ) );
 
 		$pageFields = [
-				'wid'                        => $service->getWikiId(),
-				'pageid'                     => $pageId,
-				$this->field( 'title' )      => $titleStr,
-				'titleStrict'                => $titleStr,
-				'title_em'                   => $titleStr,
-				'url'                        => $service->getUrlFromPageId( $pageId ),
-				'ns'                         => $service->getNamespaceFromPageId( $pageId ),
-				'host'                       => $service->getHostName(),
-				'lang'                       => $service->getSimpleLanguageCode(),
-				$this->field( 'wikititle' )  => $sitename,
-				'page_images'                => count( $response['parse']['images'] ),
-				'iscontent'                  => $service->isPageIdContent( $pageId ) ? 'true' : 'false',
-				'is_main_page'               => $service->isPageIdMainPage( $pageId ) ? 'true' : 'false',
-				'indexed'                    => gmdate("Y-m-d\TH:i:s\Z")
+			'wid' => $service->getWikiId(),
+			'pageid' => $pageId,
+			$this->field( 'title' ) => $titleStr,
+			'titleStrict' => $titleStr,
+			'title_em' => $titleStr,
+			'url' => $service->getUrlFromPageId( $pageId ),
+			'ns' => $service->getNamespaceFromPageId( $pageId ),
+			'host' => $service->getHostName(),
+			'lang' => $service->getSimpleLanguageCode(),
+			$this->field( 'wikititle' ) => $sitename,
+			'page_images' => count( $response['parse']['images'] ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "parse" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)